Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.opensuse.org/opensuse/leap:latest
```

Comment:
can you elaborate reasons for choosing such base ? there is well maintained python3 docker images . don't see good reason to go one level deeper and take this burden on our shelters ...

Reply:
This happened for two reasons: First, the `python3` container had some linker issues, which could be solved by using the python38 or python36 container (I don't remember which one worked). Second: I believe `pcw` would be a prime candidate to use our new and shiny BCI containers, once they are available. To make this transition happen as smooth as possible, I believe switching to leap could be a good step towards this direction.

Reply:
ok make sense